projects/packages/videopress/composer.json

```
@@ -0,0 +1,46 @@
+{
+	"name": "automattic/jetpack-videopress",
+	"description": "VideoPress package",
+	"type": "library",
```

```suggestion
	"type": "jetpack-library",
```
Opened up https://github.com/Automattic/jetpack/pull/24964 to make sure this doesn't happen with future generated packages.